src/accessControlClientInterfaces/GroupClient.ts

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import type { AccessToken } from "@itwin/core-bentley";
+import type { BentleyAPIResponse } from "../types/CommonApiTypes";
+import type { Group, MultipleGroupsResponse, SingleGroupResponse } from "../types/Groups";
+
+export interface IGroupsClient {
+  /** Retrieves a list of groups the for a specified iTwin */
+  getITwinGroups(
+    accessToken: AccessToken,
+    iTwinId: string,
+    additionalHeaders?: Record<string, string>
```

Comment:
Is there a reason that we only allow `additionalHeaders` on this one method and not the others?

Reply:
@CalebGerman lets not remove it just yet. We might still need it. I'm not sure if the RBAC Portal can support the proposed solution.

@inanayakkara Does the RBAC Portal use pnpm? Could this be a viable solution?